frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/capacity-and-nodes-step.tsx

```diff
@@ -174,6 +208,15 @@ const SelectedCapacityAndNodes: React.FC<SelectedCapacityAndNodesProps> = ({
           <GridItem span={7} />
         </Grid>
       </FormGroup>
+      {hasStrechClusterEnabled && (
+        <StretchCluster
+          enableArbiter={enableArbiter}
+          arbiterLocation={arbiterLocation}
+          zones={zones}
+          onChecked={onArbiterChecked}
+          onSelect={onZonesSelect}
+        />
+      )}
```

Comment:
Shouldn't it be above `Selected Capacity` ??

Reply:
Nope, design changes https://marvelapp.com/prototype/g1h86ga/screen/78847968